README.md

```diff
@@ -654,6 +654,74 @@ instance.delete()
 
 **Do not use delete() in production!**
 
+Bulk-updating sandboxes
+-----------------------
+
+OpenCraft IM provides a tagging mechanism that you can use to update instances in bulk.
+The following script shows how to update multiple instances at once from the Django shell:
+
+```python
+# Author: Brandon DeRosier <bdero@opencraft.com>
```

Comment:
Alright, can we list these things precisely, so that i can become a todo list for the next iterations? And use the opportunity/iteration to discuss the approach, as part of the current review? I'd like to keep this work as part of the deployment tasks, rather than something separate. It would also be useful to have your feedback from this iteration - what did take you time? What was missing in terms of automation?

> I would prefer not to make this a management command. There are often slightly different things that we need to do with each mass re-deployment, and I think its useful to be able to customize the behavior each time.

If there are things that are different each time, can we cleanly separate them from the workflow of rebuilding all the instances?